addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -143,15 +143,15 @@ private void sendDiscover(InetAddress destIP) {
             try {
                 datagramSocket.send(discoverPacket_v3);
             } catch (IOException e) {
-                logger.error("Sending a V3 discovery packet to " + destIP.getHostAddress() + " failed. "
-                        + e.getLocalizedMessage());
+                logger.error("Sending a V3 discovery packet to {} failed. {}", destIP.getHostAddress(),
+                        e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace